roles/container-engine/containerd/tasks/containerd_repo.yml

```diff
@@ -11,28 +10,14 @@
   delay: "{{ retry_stagger | d(3) }}"
   with_items: "{{ containerd_repo_key_info.repo_keys }}"
   environment: "{{ proxy_env }}"
-  when:
-    - ansible_os_family in ['Ubuntu', 'Debian']
+  when: ansible_os_family == 'Debian'
```

Comment:
Why here should support Debian only instead of both Debian and Ubuntu supports?
Is the reason Ubuntu is just an alias of Debian on Ansible?
If so, it is better to separate PRs for different purposes.

Reply:
This is the difference between ansible_os_family and ansible_distribution, this is trivial IMO but I can put better explanations in the commit message

Reply:
https://github.com/ansible/ansible/blob/20509b65071291ab6cbe8a279c734902fb4e8383/lib/ansible/module_utils/facts/system/distribution.py#L488
I just saw that in Ansible 2.10+ `Flatcar Container Linux by Kinvolk` will become `Flatcar` (https://github.com/ansible/ansible/pull/69627)

Reply:
Switched to `ansible_pkg_mgr == 'apt'`